src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -141,6 +141,10 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
                     }
                 }
             }
+            else if (!_options.IgnoreMissingFeatures)
+            {
+                throw new FeatureManagementException(FeatureManagementError.MissingFeature, $"The feature declaration for specified feature '{feature}' was not found.");
+            }
```

Comment:
We should log a warning even if `IgnoreMissingFeatures` is `true`. Suggest change below
```cs
else
{
    string errorMessage = $"The feature declaration for specified feature '{feature}' was not found.";
    if (!_options.IgnoreMissingFeatures)
    {
        throw new FeatureManagementException(FeatureManagementError.MissingFeature, errorMessage);
    }
    else
    {
        _logger.LogWarning(errorMessage);
    }
}
```